ern-api-gen/test/api-test.js

```diff
@@ -26,47 +27,23 @@ describe('api schemas', function() {
     })
   }
 
-  const generateObj = function(config) {
```

Comment:
this fn provided a mock `.then` impl.  no consumers in this file ever chained onto `.then`, and i actually needed a proper promise to test the _invalid_ cases.  thus, below, you will see i simply _removed_ the mocked promise chain, use a proper promise chain, and add real assertions. previously, there we're no assertions, just passes on no errors.  git diff here makes this look more wonky than it was